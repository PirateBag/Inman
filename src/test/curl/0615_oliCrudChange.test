Change quantity of Order #2 from 1 to 5.

===JSON===

{ "rows" : [
        {  "activityState" : "CHANGE", "id" : 2, "itemId" : "2", "quantityOrdered" : "5", "quantityAssigned" : "0.0",
        "startDate" : "2025-0711", "completeDate" : "2025-0715", "orderState" : "PLANNED",
        "debitCreditIndicator" : "ADDS_TO_BALANCE" }
] }