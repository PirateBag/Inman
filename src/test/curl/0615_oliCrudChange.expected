{
  "responseType" : "MULTILINE",
  "errors" : [ ],
  "data" : [ {
    "id" : 2,
    "crudAction" : "CHANGE",
    "itemId" : 2,
    "quantityOrdered" : 5.0,
    "quantityAssigned" : 0.0,
    "startDate" : "2025-0711",
    "completeDate" : "2025-0715",
    "parentOliId" : 0,
    "orderState" : "PLANNED",
    "orderType" : "MOHEAD"
  } ]
}